src/set_theory/cardinal/basic.lean

```diff
@@ -541,7 +547,6 @@ protected theorem lt_wf : @well_founded cardinal.{u} (<) :=
 end⟩
 
 instance : has_well_founded cardinal.{u} := ⟨(<), cardinal.lt_wf⟩
-instance : well_founded_lt cardinal.{u} := ⟨cardinal.lt_wf⟩
```

Comment:
I assume this is removed because `by apply_instance` finds it?

Reply:
Yes, it comes from the next instance.